src/components/HowToThinkAboutRisk.vue

```diff
@@ -4,28 +4,42 @@
     <h5>Click to learn more</h5>
     <v-container>
       <v-row>
-        <v-col cols="6" md="3">
+        <v-col cols="6" md="3" class="howToThinkAboutRisk__icon-container">
           <CrowdingIcon
             class="componentIcon"
             @click="onComponentIconClick('crowding')"
           ></CrowdingIcon>
-          <div class="componentLabel">Crowding</div>
+          <div @click="onComponentIconClick('crowding')" class="componentLabel">
```

Comment:
do you mind moving the @click up to the parent of the icon and the label so we don't have to assign the click handler twice?

Reply:
Sure thing, just updated it!